src/components/dashboard/EntityStatOverview/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { Stack, Typography } from '@mui/material';
+import { authenticatedRoutes } from 'app/routes';
+import LinkWrapper from 'components/shared/LinkWrapper';
+import { ReactElement } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { Entity } from 'types';
+import ActiveEntityCount from './ActiveEntityCount';
+import Statistic from './Statistic';
+
+interface Props {
+    Icon: ReactElement;
+    background: { light: string; dark: string };
+    entityType: Entity;
+    monthlyUsage?: number;
+    monthlyUsageLoading?: boolean;
+}
+
+const getEntityPageURLPath = (entityType: string): string => {
+    if (entityType === 'collection') {
+        return authenticatedRoutes.collections.fullPath;
+    }
+
+    if (entityType === 'capture') {
+        return authenticatedRoutes.captures.fullPath;
+    }
+
+    return authenticatedRoutes.materializations.fullPath;
+};
+
+const getTitleId = (entityType: string): string => {
+    if (entityType === 'collection') {
+        return 'terms.collections';
+    }
+
+    if (entityType === 'capture') {
+        return 'terms.sources';
+    }
+
+    return 'terms.destinations';
+};
```

Comment:
Don't think we need to change anything... but this initially just _feels_ odd to me but  I do not know why. I'll process it a but more to try to figure out why exactly.

Reply:
The routing-related ambiguous discomfort was discussed on a VC and this was deemed _okay for now_.